Failed to create a Subapplication via menu when Application Editor breadcrumb is selected

If the Application Editor breadcrumb is selected and then via the menu Source -> New subapplication is clicked, the error appears.

Command 'org.eclipse.fordiac.ide.application.commands.newSubApp' failed
Error executing 'org.eclipse.fordiac.ide.application.commands.newSubApp': java.lang.NullPointerException: Cannot read field "x" because "point" is null
  org.eclipse.core.commands.ExecutionException: Error executing 'org.eclipse.fordiac.ide.application.commands.newSubApp': java.lang.NullPointerException: Cannot read field "x" because "point" is null
  	at org.eclipse.e4.core.commands.internal.HandlerServiceHandler.execute(HandlerServiceHandler.java:179)
  	at org.eclipse.core.commands.Command.executeWithChecks(Command.java:488)
  	at org.eclipse.core.commands.ParameterizedCommand.executeWithChecks(ParameterizedCommand.java:485)
  	at org.eclipse.e4.core.commands.internal.HandlerServiceImpl.executeHandler(HandlerServiceImpl.java:204)
  	at org.eclipse.e4.ui.workbench.renderers.swt.HandledContributionItem.executeItem(HandledContributionItem.java:444)
  	at org.eclipse.e4.ui.workbench.renderers.swt.AbstractContributionItem.handleWidgetSelection(AbstractContributionItem.java:442)
  	at org.eclipse.e4.ui.workbench.renderers.swt.AbstractContributionItem.lambda$2(AbstractContributionItem.java:464)
  	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:91)
  	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:5855)
  	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1617)
  	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:5065)
  	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:4517)
  	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1151)
  	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:339)
  	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1042)
  	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:152)
  	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:639)
  	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:339)
  	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:546)
  	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:173)
  	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
  	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:208)
  	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:143)
  	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:109)
  	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:439)
  	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:271)
  	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
  	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
  	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:668)
  	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:605)
  	at org.eclipse.equinox.launcher.Main.run(Main.java:1481)
  	at org.eclipse.equinox.launcher.Main.main(Main.java:1454)
  Caused by: org.eclipse.e4.core.di.InjectionException: java.lang.NullPointerException: Cannot read field "x" because "point" is null
  	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:68)
  	at org.eclipse.e4.core.internal.di.InjectorImpl.invokeUsingClass(InjectorImpl.java:299)
  	at org.eclipse.e4.core.internal.di.InjectorImpl.invoke(InjectorImpl.java:233)
  	at org.eclipse.e4.core.contexts.ContextInjectionFactory.invoke(ContextInjectionFactory.java:174)
  	at org.eclipse.e4.core.commands.internal.HandlerServiceHandler.execute(HandlerServiceHandler.java:165)
  	... 31 more
  Caused by: java.lang.NullPointerException: Cannot read field "x" because "point" is null
  	at org.eclipse.fordiac.ide.application.handlers.AbstractContainerElementHandler.getEmptyContainerPosSizeRef(AbstractContainerElementHandler.java:171)
  	at org.eclipse.fordiac.ide.application.handlers.AbstractContainerElementHandler.getPosSizeRef(AbstractContainerElementHandler.java:145)
  	at org.eclipse.fordiac.ide.application.handlers.NewSubApplication.getPosSizeRef(NewSubApplication.java:1)
  	at org.eclipse.fordiac.ide.application.handlers.AbstractContainerElementHandler.execute(AbstractContainerElementHandler.java:69)
  	at org.eclipse.fordiac.ide.application.handlers.NewSubApplication.execute(NewSubApplication.java:1)
  	at org.eclipse.ui.internal.handlers.HandlerProxy.execute(HandlerProxy.java:277)
  	at org.eclipse.ui.internal.handlers.E4HandlerProxy.execute(E4HandlerProxy.java:98)
  	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
  	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
  	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:58)
  	... 35 more

--- a/plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/handlers/AbstractContainerElementHandler.java
+++ b/plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/handlers/AbstractContainerElementHandler.java
@@ -22,6 +22,7 @@ import static org.eclipse.fordiac.ide.model.ui.editors.HandlerHelper.getViewer;
 import static org.eclipse.fordiac.ide.model.ui.editors.HandlerHelper.selectElement;
 
 import java.util.List;
+import java.util.Optional;
 
 import org.eclipse.core.commands.AbstractHandler;
 import org.eclipse.core.commands.ExecutionEvent;
@@ -166,8 +167,9 @@ abstract class AbstractContainerElementHandler extends AbstractHandler {
 	protected Rectangle getEmptyContainerPosSizeRef(final ExecutionEvent event, final EditPartViewer viewer,
 			final FBNetwork network) {
 		Rectangle selectionExtend;
-		final org.eclipse.swt.graphics.Point point = ((FBNetworkContextMenuProvider) viewer.getContextMenu())
-				.getPoint();
+		final org.eclipse.swt.graphics.Point point = Optional
+				.ofNullable(((FBNetworkContextMenuProvider) viewer.getContextMenu()).getPoint())
+				.orElse(new org.eclipse.swt.graphics.Point(0, 0));
 		selectionExtend = new Rectangle(point.x, point.y, 200, 100);
 		final IFigure targetFigure = getTargetFigure(event, viewer, network);
 		targetFigure.translateToRelative(selectionExtend);

